imports/startup/client/listenActiveUsers.js

```diff
@@ -0,0 +1,48 @@
+import { Meteor } from 'meteor/meteor';
+import { Tracker } from 'meteor/tracker';
+
+const updateUser = (user, force = false) => {
+	// do not update my own user, my user's status will come from a subscription
+	if (user._id === Meteor.userId()) {
+		return;
+	}
+	if (force) {
+		return Meteor.users._collection.upsert({ _id: user._id }, {
+			$set: {
+				username: user.username,
+				name: user.name,
+				utcOffset: user.utcOffset,
+				status: user.status,
+			},
+		});
+	}
+
+	const found = Meteor.users.findOne(user._id, { fields: { _id: 1 } });
```

Comment:
Define fields is not performatic on client

Reply:
I would thought but I gave it a try, here is the results:
![image](https://user-images.githubusercontent.com/8591547/49674497-5b900a80-fa59-11e8-9655-10fc30e5ab44.png)